packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -46,34 +46,37 @@ module.exports = {
     //  port: 8545,            // Standard Ethereum port (default: none)
     //  network_id: "*",       // Any network (default: none)
     // },
-    // Another network with more advanced options...
+    //
+    // An additional network, but with some advanced options…
     // advanced: {
-    // port: 8777,             // Custom port
-    // network_id: 1342,       // Custom network
-    // gas: 8500000,           // Gas sent with each transaction (default: ~6700000)
-    // gasPrice: 20000000000,  // 20 gwei (in wei) (default: 100 gwei)
-    // from: <address>,        // Account to send txs from (default: accounts[0])
-    // websocket: true        // Enable EventEmitter interface for web3 (default: false)
+    //   port: 8777,             // Custom port
+    //   network_id: 1342,       // Custom network
+    //   gas: 8500000,           // Gas sent with each transaction (default: ~6700000)
+    //   gasPrice: 20000000000,  // 20 gwei (in wei) (default: 100 gwei)
+    //   from: <address>,        // Account to send transactions from (default: accounts[0])
+    //   websocket: true         // Enable EventEmitter interface for web3 (default: false)
     // },
+    //
     // Useful for deploying to a public network.
-    // NB: It's important to wrap the provider as a function.
+    // Note: It's important to wrap the provider as a function to ensure truffle uses a new provider every time.
```

Comment:
Uses a new provider every time is ambiguous. I would rather leave it as the suggestion below and expand the reasoning in our documentation. 

```suggestion
    // Note: It's important to wrap the provider as a function.
```